charts/operator-wandb/charts/clickhouse/templates/keeper_statefulset.yaml

```diff
@@ -0,0 +1,139 @@
+  podManagementPolicy: "{{ .Values.common.podManagementPolicy }}"
+  selector:
+    matchLabels:
+      {{ include "wandb.selectorLabels" $ | nindent 6 }}
+      {{ include "clickhouse.labels" . | nindent 6 }}
+  serviceName: {{ include "clickhouse.fullname" . }}-ch-keeper-headless
+  template:
+    metadata:
+      labels:
+        {{ include "wandb.podLabels" . | nindent 8 }}
+        {{ include "clickhouse.commonLabels" . | nindent 8 }}
+        {{ include "clickhouse.podLabels" . | nindent 8 }}
+        {{ include "clickhouse.labels" . | nindent 8 }}
+      annotations:
+        {{- if .Values.pod.annotations -}}
+        {{-   toYaml .Values.pod.annotations | nindent 8 }}
+        {{- end }}
+    spec:
+      imagePullSecrets:
+      {{- include "wandb.imagePullSecrets" . }}
+      serviceAccountName: {{ .Values.serviceAccount.name }}
+      automountServiceAccountToken: {{ .Values.serviceAccount.automountServiceAccountToken }}
+      securityContext:
+        fsGroup: 1000710000
+        fsGroupChangePolicy: Always
+      containers:
+        - name: keeper
+          image: "{{ .Values.keeper_image.repository }}:{{ .Values.keeper_image.tag }}"
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: CLICKHOUSE_USER
+              value: "default"
+            - name: CLICKHOUSE_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "wandb.clickhouse.passwordSecret" . }}
+                  key: CLICKHOUSE_PASSWORD
+          volumeMounts:
+            - name: processed-config
+              mountPath: /etc/clickhouse-keeper
+            - name: keeper-data
+              mountPath: /var/lib/clickhouse
+            - name: keeper-data
+              mountPath: /var/lib/clickhouse-keeper
+            - name: keeper-data
+              mountPath: /var/log/clickhouse-keeper
+            - name: config-volume
+              mountPath: /etc/clickhouse-keeper/keeper_config.xml
+              subPathExpr: "$(POD_NAME).xml"
+              readOnly: true
+          securityContext:
+            allowPrivilegeEscalation: {{ .Values.common.containerSecurityContext.allowPrivilegeEscalation }}
+            capabilities:
+              drop:
+                - {{ .Values.common.containerSecurityContext.capabilities.drop | join "\n                - " }}
+            privileged: {{ .Values.common.containerSecurityContext.privileged }}
+            readOnlyRootFilesystem: {{ .Values.common.containerSecurityContext.readOnlyRootFilesystem }}
+            runAsGroup: 1000710000
+            runAsNonRoot: true
+            runAsUser: 1000710000
+            seLinuxOptions:
+              level: "{{ .Values.common.containerSecurityContext.seLinuxOptions.level }}"
+            seccompProfile:
+              type: {{ .Values.common.containerSecurityContext.seccompProfile.type }}
```

Comment:
Please refer to the comment about `securityContext` in the Click House server config